lib/google_scraping_web/templates/keyword/show.html.heex

```diff
@@ -1,3 +1,73 @@
-<h2>Keywords details <%= @keyword.name %></h2>
+<nav aria-label="breadcrumb">
+  <ol class="breadcrumb">
+    <li class="breadcrumb-item">
+      <%= link(gettext("Keywords"),
+        to: Routes.keyword_path(@conn, :index)
+      ) %>
+    </li>
+    <li class="breadcrumb-item active" aria-current="page">
+      <%= gettext("Keyword '%{name}' details", %{name: @keyword.name}) %>
+    </li>
+  </ol>
+</nav>
 
-<p>Coming soon!</p>
+<div class="card">
+  <div class="card-header text-center">
+    <h2><%= @keyword.name %></h2>
+  </div>
+  <div class="card-body">
+    <dl class="row">
+      <dt class="offset-sm-3 col-lg-5"><%= gettext("Status") %></dt>
+      <dd class="col-lg-4"><%= @keyword.status %></dd>
+      <%= if @keyword.status == :completed do %>
```

Comment:
We are checking this multiple times in this file, how about moving it as a function in a view file? may be `KeywordView`?

Reply:
Thanks. It was done here: https://github.com/Nihisil/nimble-certification-elixir/pull/50/commits/cb8fae2af936be3c641edc3f95e65b2d9f5d2675